src/platform/testing/e2e/helpers.js

```diff
@@ -118,6 +118,10 @@ function expectNavigateAwayFrom(client, urlSubstring) {
     .before(Timeouts.slow);
 }
 
+function expectElementNotPresent(client, elementSelector) {
+  client.expect.element(elementSelector).to.not.be.present;
+}
+
```

Comment:
I'm not seeing this used anywhere - is it necessary?

Reply:
I have no idea, I actually didn't add this. no idea how it is in this branch at all.

Reply:
have you merged master recently?

Reply:
It appears that you did add it as part of the commit ["added stuff"](https://github.com/department-of-veterans-affairs/vets-website/pull/12995/commits/45780eadfab65faddfaea8fb0edb946e385bcd84)

Reply:
@rianfowler Just merged in

Reply:
@bkjohnson so do I just need to remove it and push?

Reply:
Yes.

Reply:
@bkjohnson removed and pushed

Reply:
Cool.  It looks like you can merge if you want since a frontend review isn't required.